static/app/components/forms/controls/radioGroup.tsx

```diff
@@ -52,12 +52,7 @@ const RadioGroup = <C extends string>({
   orientInline,
   ...props
 }: RadioGroupProps<C>) => (
-  <Container
-    orientInline={orientInline}
-    {...props}
-    role="radiogroup"
-    aria-labelledby={label}
```

Comment:
why was this wrong? mind adding a description? 🙏

Reply:
Updated the description. `aria-labelledby` is supposed to be given an ID, which isn't what it was getting.